src/components/footer/index.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Icon } from './icon';
+
+export const Footer = (): JSX.Element => {
+    return (
+        <footer className="pv4 ph3 ph5-ns tc">
+            <div className={'flex flex-column ma4'}>
+                <div>
+                    <Icon icon={'meetup'} url={'https://www.meetup.com/toronto-software-development-meetup-group/'}></Icon>
+                    <Icon icon={'twitter'} url={'https://twitter.com/wwgtoronto'}></Icon>
+                    <Icon icon={'envelope'} url={'mailto:wwgtoronto@gmail.com'}></Icon>
+                    <Icon icon={'slack'} url={'#'}></Icon>
+                    <Icon icon={'github'} url={'https://github.com/WWGToronto'}></Icon>
+                </div>
+            </div>
+            <div className="mt4">
+                <a href="/" className="f6 link dim gray dib mr3 mr4-ns">Home</a>
+                <a href="/events" className="f6 link dim gray dib mr3 mr4-ns">Events</a>
+                <a href="/sponsors" className="f6 link dim gray dib mr3 mr4-ns">Sponsors</a>
+                <a href="#" className="f6 link dim gray dib">Code of Conduct</a>
```

Comment:
Where is this going to link to? CoC on meetup or a new page?

Reply:
At the time I was thinking of linking to our CoC on meetup as a short-term solution. In the long-term, perhaps the Code of Conduct would be housed here on our website, for membership to our club in general. Then we can put the link on event promotion websites not limited to meetup (eventbrite, etc).